test/integration/acceptance/remove_signatory_test.cpp

```diff
@@ -0,0 +1,212 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
```

Comment:
I agree that this is a debatable question.
Here we use acceptance_fixture.hpp that definitely uses gtest due to inheritance from `::testing::Test`. Thus, gtest include is redundant here.

Reply:
There's a principle https://include-what-you-use.org/ (whoah, there's even a website for that)
So, for example, if the AcceptanceFixture for some reason won't rely on gtest, the test need to be fixed. That barely possible, though it's not a big deal to leave one more line, rather than combining all of the headers in your mind